lib/datadog/security/assets/waf_rules.json

```diff
@@ -0,0 +1,3532 @@
+{
```

Comment:
This file is a dump of the WAF rules we used at Sqreen. FYI on the AppSec side we have a new format in development to tackle some limitations of the previous one. The process of updating the file is still unclear for now, I'll keep you updated on that as it evolves. For now this will be a static asset.

There might be rules irrelevant for Ruby but that can still trigger detection of bad actors (e.g a PHP oriented attack hitting a Ruby app can still help identify a bad actor)

Reply:
The current canonical set of rules in the new format is [over there](https://github.com/DataDog/appsec-event-rules/blob/main/build/recommended.yaml)

Reply:
Thanks for the clarification! Can I convince you to add a `README.waf_rules.md` or something like that with the details you shared, so that they "stay close" to the file itself and provide that context? :)

Reply:
Good point!

Reply:
Addressed